tests/BlazingSQLTest/EndToEndTests/allE2ETest-v2.py

```diff
@@ -0,0 +1,204 @@
+from Configuration import ExecutionMode
+from Configuration import Settings
+
+from DataBase import createSchema
+from pynvml import nvmlInit
+from Utils import Execution, init_context
+from blazingsql import DataType
+from Runner import runTest
+from Runner import testSuites
+import sys
+import time
+
+def initializeDrill():
+    # Create Table Drill -----------------------------------------
+    from pydrill.client import PyDrill
+
+    drill = PyDrill(host="localhost", port=8047)
+    createSchema.init_drill_schema(
```

Comment:
Lets make initializing for the different data sets a little more parametric, so that we dont have a bool variable for every different data set, and so that its cleaner and easier to add more data sets in the future.

createSchema.init_drill_schema(
        drill, Settings.data["TestSettings"]["dataDirectory"], test_sets={'tpch':'parquet' , 'bool_test':'csv' , 'smiles_test':'parquet'})


OR perhaps even better is that these test sets and how they are to be loaded is configured in a yaml

Reply:
This can be done in the following PR